cli/pkg/cmd/create/helm.go

```diff
@@ -78,10 +78,11 @@ func (h *Helm) Setup() error {
 }
 
 func (h *Helm) InstallCharts() error {
+	chartURL := "https://airy-core-helm-charts.s3.amazonaws.com/stable/airy-" + h.version + ".tgz"
```

Comment:
Instead of building our own image, a generic `alpine/helm:3.6.3` image is used with an URL with the appropriate version of the charts.